.github/workflows/ci.yml

```
@@ -188,6 +198,19 @@ jobs:
           UNITY_VERSION: ${{ matrix.unity-version }}
         run: dotnet msbuild /t:UnitySmokeTestStandalonePlayerIL2CPP
 
+      - name: Setup Android Emulators
+        if: matrix.os == 'macos-latest'
+        uses: reactivecircus/android-emulator-runner@v2
```

Not a verified action. Please pin this to a commit hash:

```suggestion
        uses: reactivecircus/android-emulator-runner@9216c92cd9935b2aea47751a50e686a9c884b2dc
```